css/cmubuggy-bootstrap.css

```diff
@@ -2330,9 +2132,9 @@ select.form-control-lg:not([size]):not([multiple]), .input-group-lg > select.for
   white-space: nowrap;
   vertical-align: middle;
   -webkit-user-select: none;
-     -moz-user-select: none;
-      -ms-user-select: none;
-          user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
```

Comment:
Are we able to kill this with the other `ms` items?

Reply:
Same answer: Autogenerated file that updated a bunch of old vendor prefixes.